syne_tune/blackbox_repository/blackbox_surrogate.py

```diff
@@ -12,25 +12,19 @@
 # permissions and limitations under the License.
 from typing import Optional, Dict
 import pandas as pd
-from sklearn.neighbors import KNeighborsRegressor
-from sklearn.pipeline import Pipeline, FeatureUnion, make_pipeline
-from sklearn.preprocessing import OneHotEncoder, StandardScaler
-from sklearn.base import BaseEstimator, TransformerMixin
```

Comment:
Moving all imports into functions should be avoided (see [PEP8](https://peps.python.org/pep-0008/)), could you revert the changes that move import into functions there and bellow?